src/utils/prismImport.js

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable filenames/match-regex */
+
+import 'prismjs/components/prism-core';
```

Comment:
How much weight does this add to be part of the bundle seeing that we're including "everything"?

Reply:
All in all if we take the other React-live PR into account it’s a net minus altogether:

> before the react-live and codepane refactors: 2.4M (min, non-gzip, without prism) and *after:* 1.6M (min, non-gzip, with prism)

Reply:
How much is just the prism part if you "external" it?

Reply:
I’ll check

Reply:
@ryan-roemer so, it's tough to exclude prism without excluding buble as both are also imported by react-live with the respective prism subpaths as well, but if I exclude: prism, react-live, and (thus also) buble, then I we land at ~1.33MB

Reply:
So that's just a `~0.3MB` min difference? If so, I'm totally cool with that...